src/pybind/mgr/dashboard/frontend/src/app/core/layouts/workbench-layout/workbench-layout.component.scss

```diff
@@ -10,7 +10,7 @@
   overflow: auto;
 }
 
-::ng-deep #toast-container {
+::ng-deep cd-workbench-layout #toast-container {
```

Comment:
This block of styles is exactly the same in `blank-layout.component.scss` and both don't work with this solution.
We should remove it from both files and put it in `src/styles/ceph-custom/_toast.scss`